docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - media_volume:/experimenter/experimenter/media
       - /experimenter/experimenter/legacy/legacy-ui/core/.cache/
       - /experimenter/experimenter/legacy/legacy-ui/core/node_modules/
-      - /experimenter/experimenter/theme/static_src/node_modules/
+      - /experimenter/experimenter/nimbus_ui_new/theme/static_src/node_modules/
       - /experimenter/experimenter/nimbus-ui/node_modules/
```

Comment:
nimbus_ui_new vs nimbus-ui ?

Reply:
I know it's because `-` is not valid for Python package names, which didn't matter for `nimbus-ui` but does for `nimbus_ui_new` since it has views, tests, etc in it.  Annoying.